### ISSUE TITLE
Add proper heading levels (accessibility)

### ISSUE THREAD

**Issue by mejackreed** [sul-dlss/earthworks#174]
- [x] home page
- [x] results page
- [x] show page

**Comment by mejackreed** [comment]
Addressing upstream in https://github.com/projectblacklight/blacklight/pull/1124
